R/parse-body.R

```diff
@@ -564,6 +574,7 @@ register_parsers_onLoad <- function() {
   # yaml types: https://stackoverflow.com/a/38000954/591574
   register_parser("yaml",    parser_yaml,    fixed = c("text/vnd.yaml", "application/yaml", "application/x-yaml", "text/yaml", "text/x-yaml"))
   register_parser("none",    parser_none,    regex = "*")
+  register_parser("geojson", parser_geojson, fixed = c("application/geo+json", "geo+json"))
```

Comment:
Should we also add `application/vnd.geo+json`?

Removing invalid content type. (Must have `text` or `application` to start)
```suggestion
  register_parser("geojson", parser_geojson, fixed = c("application/geo+json"))
```